userguide/tutorials/plugin_installation.adoc

```diff
@@ -167,7 +168,25 @@ curl -v \
 
 === Deploying by Hand
 
-Deploying by hand consists of building the self contained OSGI jar, and copying that jar at the right location. For example, the `adyen` plugin with a version with version `0.3.2` would show up as the following:
+As explained in link:#_overview[overview section], Kill Bill have 2 locations to deploying plugins:
+
+1. `/platform` (bundles) for Kill Bill internal plugins, and;
+2. `/plugins` for 3rd party plugins.
+
+Deploying bundles is straightforward:
+
+1. Build `killbill-platform` project.
```

Comment:
Does this refer to the https://github.com/killbill/killbill-platform repo? If so, consider adding a link to the repo to make this clear.

Reply:
fixed